xcodeproj/internal/bazel_integration_files/import_indexstores.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+set -euo pipefail
+
+readonly pidfile="$OBJROOT/import_indexstores.pid"
+readonly execution_root="$1"
+shift
+
+# Kill previously running import
+if [[ -s "$pidfile" ]]; then
+  pid=$(cat "$pidfile")
+  kill "$pid" 2>/dev/null || true
+  while kill -0 "$pid" 2>/dev/null; do
+    sleep 1
+  done
+fi
+
+# Exit early if no indexstore filelists were provided
+if [ $# -eq 0 ]; then
+  exit
+fi
+
+# Set pid to allow cleanup later
+echo $$ > "$pidfile"
+trap 'rm "$pidfile" 2>/dev/null || true' EXIT
+
+# Merge all filelists into a single file
+filelist="$(mktemp)"
+sort -u "$@" | sed "s|^|$execution_root/|" > "$filelist"
+
+# Exit early if no indexstores are provided
+if [ ! -s "$filelist" ]; then
+  exit
+fi
```

Comment:
@erikkerber A refactor removed this line. This should fix your issue on rules_swift 1.1.0.